### PR TITLE
Update cluster up documentation (persistence, proxy)

### PR DESCRIPTION
**Pull request by csrwng**
Adds information about persistence and proxy settings
Adds step to Unix setup to allow container access through firewall

### CODE REVIEW

**Comment by csrwng** [comment]
@bparees ptal

File: docs/cluster_up_down.md [hunk 1]
```diff
@@ -232,6 +250,40 @@ Once the machine has been created, the `--create-machine` argument is no longer
...
+You should at least add the registry service IP (172.30.1.1) to the Docker daemon's NO_PROXY environment variable.
+How these settings are specified will vary depending on your platform. Both Docker for Windows and Docker for Mac provide a 
+settings GUI for specifying a proxy. On RHEL or Fedora, you can add HTTP_PROXY, HTTPS_PROXY, and NO_PROXY variables to 
+/etc/sysconfig/docker (or /etc/sysconfig/docker-latest depending on which service you're using).
+Cluster up will warn you if there is a discrepancy between the Docker settings and the arguments you specify for cluster up.
+
+
+| WARNING |
+| ------- |
+| On Docker for Windows, if you specify proxy settings for Docker in its GUI, Docker will apply those settings to every container that is run on the daemon. This requires that you specifically exclude any services that could be accessed container to container (172.30.0.0/16) |
```
Comment (bparees) [inline docs/cluster_up_down.md]: how do you exclude them?

Comment (csrwng) [inline docs/cluster_up_down.md]: @bparees added instructions on how to exclude addresses in Windows.
(btw, I'd completely forgotten about this pull)

**Comment by bparees** [comment]
lgtm [merge]

**Pull request merged**